charmcraft/commands/store/store.py

```diff
@@ -114,3 +118,19 @@ def upload(self, name, filepath):
             # XXX Facundo 2020-06-30: Implement a slight backoff algorithm and fallout after
             # N attempts (which should be big, as per snapcraft experience). Issue: #79.
             time.sleep(POLL_DELAY)
+
+    def list_revisions(self, name):
+        """Return charm revisions for the indicated charm."""
+        response = self._client.get('/v1/charm/{}/revisions'.format(name))
+        result = []
+        for item in response['revisions']:
+            errors = [Error(message=e['message'], code=e['code']) for e in (item['errors'] or [])]
+            result.append(Revision(
+                revision=item['revision'],
+                version=item['version'],
+                # we could use `datetime.datetime.fromisoformat` after we drop Py35 support
```

Comment:
> classmethod datetime.fromisoformat(date_string)
> [...]
>    New in version 3.7.

Reply:
Oh, right! Improved this message and removed the one in `requirements.txt`